README.md

```diff
@@ -87,6 +87,12 @@ module.exports.webpack = function(config) {
 };
 ```
 
+You can use this functionality to inject external styles or scripts if your
+components need e.g. Bootstrap to work.
+```js
+config.entry.push('../../../injectStyleTag.js');
```

Comment:
> I suppose most use cases would include external file from CDN, in which case you need a custom JS file to inject actual script/style tags.

So webpack doesn't support external files as entry points?

> Using ./injectStyleTag.js would look for the file in cosmos project inside node_modules. In this case the file injectStyleTag.js is actually in the root of my project. I agree it looks a bit odd, but I'm not sure how to get around it.

Maybe you could use `path.join(process.cwd(), 'injectStyleTag.js')`